### PR TITLE
Fixed QueryResult Property - Dialogflow

### PR DESCRIPTION
**Pull request by khalilchoudhry**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [ ] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [ ] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [ ] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [ ] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://cloud.google.com/dialogflow-enterprise/docs/reference/rpc/google.cloud.dialogflow.v2#google.cloud.dialogflow.v2.QueryResult>
- [x] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

If removing a declaration:
- [ ] If a package was never on DefinitelyTyped, you don't need to do anything. (If you wrote a package and provided types, you don't need to register it with us.)
- [ ] Delete the package's directory.
- [ ] Add it to `notNeededPackages.json`.

### CODE REVIEW

**Comment by dyladan** [comment]
@khalilchoudhry I noticed you added yourself to the authors list in the typing file headers. This is typically reserved for people who will actively maintain the typings and review pull requests. I can see that you aren't very active on GitHub and this is your first DT contribution. Did you mean to add yourself as a typings author?

File: types/dialogflow/index.d.ts [hunk 1]
```diff
@@ -1,6 +1,8 @@
-// Type definitions for dialogflow 0.9
+// Type definitions for dialogflow 1.0
```
Comment (dyladan) [inline types/dialogflow/index.d.ts]: dialogflow package is currently at 0.9.1 not 1.0

File: types/dialogflow/index.d.ts [hunk 2]
```diff
@@ -1,6 +1,8 @@
...
 // Project: https://github.com/googleapis/nodejs-dialogflow
-// Definitions by: Daniel Dyla <https://github.com/dyladan>, Tom Carrio <https://github.com/tcarrio>
+// Definitions by: Daniel Dyla <https://github.com/dyladan>
+//                 Tom Carrio <https://github.com/tcarrio>
+//                 Khalil Choudhry <https://github.com/khalilchoudhry>
```
Comment (dyladan) [inline types/dialogflow/index.d.ts]: This header is typically for maintainers, not just attribution. Did you mean to add yourself as a maintainer?

**Review by dyladan — CHANGES_REQUESTED** [review]

File: types/dialogflow/index.d.ts (hunk 1 above)
Comment (khalilchoudhry) [reply types/dialogflow/index.d.ts]: It's written on DT's readme to update to major version on breaking change. That's why I updated it to 1.0.

File: types/dialogflow/index.d.ts (hunk 2 above)
Comment (khalilchoudhry) [reply types/dialogflow/index.d.ts]: Actually I thought it's for attribution, but now, I would like to be maintainer as I actively use dialogflow's nodejs client.

**Review by dyladan — APPROVED** [review]

File: types/dialogflow/index.d.ts (hunk 2 above)
Comment (dyladan) [reply types/dialogflow/index.d.ts]: ok no problem I approved the request

**Pull request merged**